11-lorawan/README.md

```diff
@@ -188,3 +188,115 @@ ST B-L072Z-LRWAN1 or any other STM32 L0/L1 Nucleo 64 with an Mbed LoRa shield.
 
 The erase subcommand correctly erases stored parameters. All saved parameters
 are correctly reloaded after a reboot.
+
+Task #05 - GNRC LoRaWAN (Over The Air Activation)
```

Comment:
Shouldn't this already be in?

Reply:
It seems I rebased to the wrong base. Solved